### PR TITLE
Makes enough levels in preflow_push

### PR DESCRIPTION
**Pull request by boothby**
This change resolves issue #1542

### CODE REVIEW

**Comment by jfinkels** [comment]
Please add a test case with the example you gave in the referenced issue.

**Comment by boothby** [comment]
Apoligies, this is my first time contributing to networkx, and my first time doing a pull request on github.  Where do I add a test case?

**Comment by boothby** [comment]
nevermind, I got it

File: networkx/algorithms/flow/tests/test_maxflow.py [hunk 1]
```diff
@@ -434,6 +434,14 @@ def test_preflow_push_global_relabel_freq():
     assert_raises(nx.NetworkXError, preflow_push, G, 1, 2,
                   global_relabel_freq=-1)
 
+def test_preflow_push_makes_enough_space():
+    #From ticket #1542
+    G = nx.DiGraph()
+    G.add_path([0,1,3],capacity=1)
+    G.add_path([1,2,3],capacity=1)
+    R = preflow_push(G,0,3,value_only=False)
+    assert_equal(R.graph['flow_value'], 1)
```
Comment (ysitu) [inline networkx/algorithms/flow/tests/test_maxflow.py]: Please ensure that every comma is followed by a space.

**Pull request merged**